tests/test_fuzzy_dedup.py

```diff
@@ -367,3 +370,67 @@ def test_from_yaml(self, tmpdir):
         config = FuzzyDuplicatesConfig.from_yaml(tmpdir / "config.yaml")
         for param in yaml_params:
             assert getattr(config, param) == yaml_params[param]
+
+
+# TODO: This test should also work on CPU. However,
+# `shuffle_utils.py` will still try to import cudf
```

Comment:
Do we care that this test also works with pandas-backed data? If so, I can also adda `"backend"` parameter to the test.